spec/syntax.md

```diff
@@ -331,25 +394,45 @@ This serves 3 purposes:
 pattern = "{" *(text / expression) "}"
 ```
 
-> Example:
->
+> **Example**
+> 
+> A simple _pattern_ containing _text_:
 > ```
 > {Hello, world!}
 > ```
-
-Whitespace within a _pattern_ is meaningful and MUST be preserved.
+>
+> An empty _pattern_:
+> ```
+> {}
+> ```
+>
+> Some _patterns_ with _expressions_:
+> ```
+> {{$foo}}
+> {Hello {$user}!}
+> {You sent {$count :number maxFractionDigits=0} notifications to {$numFriends :number type=spellout} friends.}
+> ```
+>
+> A _pattern_ containing three spaces:
+> ```
+> {   }
+> ```
 
 ### Expressions
 
-**_Expressions_** MUST start with an _operand_ or an _annotation_.
+An _expression_ can appear as a local variable value, as a _selector_, and within a _pattern_.
+Each _expression_ MUST start with an _operand_ or an _annotation_.
```

Comment:
```suggestion
The contents of each _expression_ MUST start with an _operand_ or an _annotation_.
```

(I think there's something missing about curly brackets having to enclose an expression?)